apps/judicial-system/web/src/components/Header/Header.tsx

```diff
@@ -147,8 +148,10 @@ const HeaderContainer: React.FC = () => {
                                   markdown={formatMessage(
                                     header.headerTipDisclaimer,
                                     {
-                                      linkStart:
-                                        '<a href="mailto:gudlaug.thorhallsdottir@dmr.is" rel="noopener noreferrer nofollow" target="_blank">gudlaug.thorhallsdottir@dmr.is',
+                                      linkStart: `<a href="mailto:gudlaug.thorhallsdottir@dmr.is" rel="noopener noreferrer nofollow" target="_blank">${
```

Comment:
Should the `href="mailto:` not also be the support email?

Reply:
Yes, thank you.